src/workerd/server/workerd.capnp

```diff
@@ -352,7 +351,8 @@ struct Worker {
       # `getenv()` with that name. If the environment variable isn't set, the binding value is
       # `null`.
 
-      analyticsEngine @17 :ServiceDesignator; 
+      analyticsEngine @17 :ServiceDesignator;
+      # An Analytics Engine binding
```

Comment:
I think it would be useful to add a more elaborate comment here now that the comments below are removed.